magefile.go

```diff
@@ -519,6 +519,16 @@ func Install() {
 
 	// Copy porter binaries
 	mgx.Must(os.MkdirAll(porterHome, pkg.FileModeDirectory))
+
```

Comment:
nit: maybe we can create a function called clearCache or something to capture this hack in once place?

Reply:
I'll open something in magefiles that we could do in a later follow-up if it comes up again.

https://github.com/getporter/magefiles/issues/22